test/jdk/javax/swing/JComboBox/8019180/Test8019180.java

```diff
@@ -60,7 +60,7 @@ public void run() {
             this.test.setSelectedIndex(1 + index);
             if (0 > this.test.getSelectedIndex()) {
                 System.err.println("ERROR: no selection");
-                System.exit(8019180);
+                throw new RuntimeException("Combobox not selected");
             }
             SwingUtilities.getWindowAncestor(this.test).dispose();
             LATCH.countDown();
```

Comment:
Looks like that before the fix all tests had graceful shutdown in standalone mode.

Now there are hanging open windows when they fails.
e.g. those two lines could be wrapped in `finally` block for graceful test finish.

Reply:
I am not able to reproduce this observation. Anyway, I have added dispose in finally block

Reply:
It is because the test is not failing right now, you can change  `>` to `<` in `if (0 > this.test.getSelectedIndex()) {`  to fail the test.

It will still hang after latest modifications when launched with `java Test8019180.java` because of `LATCH` is still awaiting for its `countdown` after throwing an exception.

It is not an issue if you run it with jtreg.

Reply:
ok modified test.